rdm/audit_for_gaps.py

```diff
@@ -0,0 +1,133 @@
+import os
+
+
+def audit_for_gaps(args):
```

Comment:
It seems like it would be more decoupled (and more consistent with the other rdm code) to pass in `args.checklist` as the first parameter to `audit_for_gaps` and `args.files` as the second parameter, instead of passing in the args object.

Reply:
Fixed.